nifi-commons/nifi-record/src/main/java/org/apache/nifi/serialization/record/ResultSetRecordSet.java

```diff
@@ -350,6 +350,8 @@ private static RecordFieldType getFieldType(final int sqlType) {
                 return RecordFieldType.TIME;
             case Types.TIMESTAMP:
             case Types.TIMESTAMP_WITH_TIMEZONE:
+            case -101: // Oracle's TIMESTAMP WITH TIME ZONE
```

Comment:
@markap14 I don't think we need these cases (Calcite would never return Oracle types, and there'd be no other way to generate them?) but put them in just in case (as this does Java SQL Timestamp handling). If you agree then I can remove this (although it's likely no-harm-no-foul)

Reply:
@mattyb149 @markap14 Given how proprietary Oracle can be, I don't see how this could hurt because it's going to be difficult to say what they'll do in the future and how Calcite might respond to that.